public/delegates.json

```diff
@@ -160,5 +160,11 @@
         "url": "www.voguetensor.ai",
         "description": "Designing branded clothing for the Bittensor community.",
         "signature": "2c4079124ae0a738106a2430e2c27ad855122d4afcc487ab0158b705cd5f915f7790cdb2fdd8db899b8cbd40448d1478be71cde1b76de31945991b548cfcc084"
+    },
+    "5DvTpiniW9s3APmHRYn8FroUWyfnLtrsid5Mtn5EwMXHN2ed": {
+        "name": "FirstTensor",
+        "url": "https://firsttensor.com",
+        "description": "Powered by people for people. Maximize your TAO returns by delegating to FirstTensor as no fees are applied.",
```

Comment:
@unconst , @shibshib ?

Reply:
Hi, in order to modify accordingly, which part is not allowed exactly?
I can see some Validator's descriptions using this and they are already approved, as: 

 "Maximize your return when staking with TAO-Validator.com. TAO-Validator.com is a highly secure validator that aims to become one of the top contributing" - tao-validator
"The go to validator for maximal TAO returns to stakers." -taostation

Reply:
Hi, could someone provide an answer on the above matter? 4 days have passed without an answer. Thank you!